tests/resources/test_resources.py

```diff
@@ -105,6 +105,21 @@ def test_create_banner(client, admin, headers):
     assert banner["active"] == banner_data["active"]
 
 
+def test_disable_expired_after_create_action(client, admin, headers):
+    """Disable expired banners after a create a banner action."""
+    # create banner first
+    banner1 = BannerModel.create(banners["banner1"])
```

Comment:
I'm not sure about this but maybe as we are testing the resource we shouldn't use the model directly?